### PR TITLE
Create weston.rb

### PR DESCRIPTION
**Pull request by lovedheart**
#889

### CODE REVIEW

File: packages/weston.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    depends_on 'pango'
+    depends_on 'dbus'
+
+    
+  def self.build
+      system "./autogen.sh"  
+      system "./configure \
+              --prefix=#{CREW_PREFIX} \
+              --libdir=#{CREW_LIB_PREFIX} \
+              --disable-lcms \
```
Comment (uberhacker) [inline packages/weston.rb]: Does `depends_on 'lcms'` create an issue?

File: packages/weston.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+              --disable-systemd-notify \
+              --disable-weston-launch"    
+      system "make"
+  end
+
+  def self.install
+      system "make DESTDIR=#{CREW_DEST_DIR} install"
+  end
```
Comment (uberhacker) [inline packages/weston.rb]: Please use 2 space instead of 4 space indentation.

**Comment by uberhacker** [comment]
Thank you for the new package @lovedheart.  Please see inline comments.

File: packages/weston.rb [hunk 3]
```diff
@@ -3,34 +3,33 @@
...
-              --libdir=#{CREW_LIB_PREFIX} \
-              --disable-lcms \
-              --disable-systemd-notify \
-              --disable-weston-launch"    
-      system "make"
+    system "./autogen.sh"  
+    system "./configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            --disable-weston-launch"
```
Comment (uberhacker) [inline packages/weston.rb]: Since you took out `--disable-lcms`, do you need `depends_on 'lcms'`?

Comment (lovedheart) [reply packages/weston.rb]: No. I have checked it.

**Pull request merged**